hazelcast/src/test/resources/hazelcast-fullconfig-advanced-network-config.yaml

```diff
@@ -0,0 +1,225 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# A comprehensive example of Hazelcast configuration with advanced network configuration.
+#
+# Hazelcast resolves configuration using the following approach:
+#
+# 1. First is checks to see if the ’hazelcast.config’ system property is set. If it is, then the value is used as the path.
+#
+#    The config option can be set by adding the following to the java command: -Dhazelcast.config=path_to_the_hazelcast.yaml.
+#
+#    The value can be a normal file path, but can also be a classpath reference if it is prefixed with ’classpath:’.
+#
+# 2. Otherwise it checks if there is a ’hazelcast.yaml’ in the working directory.
+#
+# 3. After that it checks if there is a ’hazelcast.yaml’ in the root of the classpath.
+#
+# 4. If a configuration cannot be found, Hazelcast will use the default hazelcast configuration
+#    ’hazelcast-default.yaml’, which is included in the the Hazelcast jar
+
+
+hazelcast:
+
+  import:
+    - hazelcast-fullconfig-without-network.yaml
+
+  advanced-network:
+    enabled: true
+    join:
+      multicast:
+        enabled: true
+        loopbackModeEnabled: true
+        multicast-group: 1.2.3.4
+        multicast-port: 12345
+        multicast-timeout-seconds: 5
+        multicast-time-to-live: 10
+        trusted-interfaces:
+          - 10.10.1.*
+          - 10.10.2.*
+      tcp-ip:
+        enabled: false
+        connection-timeout-seconds: 123
+        required-member: dummy
+        member: dummy1,dummy2
+        interface: 127.0.0.10
+        members: dummy3,dummy4
+        member-list:
+          - dummy5
+          - dummy6
+      aws:
+        enabled: false
+        access-key: my-access-key
+        secret-key: my-secret-key
+        iam-role: dummy
+        # optional, default is us-east-1
+        region: us-west-1
+        # optional, default is ec2.amazonaws.com. If set, region shouldn't be set as it will override this property
+        host-header: ec2.amazonaws.com
+        # optional, only instances belonging to this group will be discovered, default will try all running instances
+        security-group-name: hazelcast-sg
+        tag-key: type
+        tag-value: hz-nodes
+        use-public-ip: true
+      gcp:
+        enabled: false
+        zones: us-east1-b,us-east1-c
+      azure:
+        enabled: false
+        client-id: CLIENT_ID
+        client-secret: CLIENT_SECRET
+        tenant-id: TENANT_ID
+        subscription-id: SUB_ID
+        cluster-id: HZLCAST001
+        group-name: GROUP-NAME
+      kubernetes:
+        enabled: false
+        namespace: MY-KUBERNETES-NAMESPACE
+        service-name: MY-SERVICE-NAME
+        service-label-name: MY-SERVICE-LABEL-NAME
+        service-label-value: MY-SERVICE-LABEL-VALUE
+      eureka:
+        enabled: false
+        self-registration: true
+        namespace: hazelcast
+#       <discovery-strategies>
```

Comment:
Should this commented out block be here? If yes, it's probably better to rewrite it in YAML.

Reply:
Good catch. Will fix in a separate PR if no other need for updating this.